src/main/java/org/folio/linked/data/service/impl/ResourceServiceImpl.java

```diff
@@ -68,16 +69,18 @@ public ResourceResponseDto createResource(ResourceRequestDto resourceDto) {
     log.info("createResource\n[{}]\nfrom DTO [{}]", mapped, resourceDto);
     metadataService.ensure(mapped);
     var persisted = saveMergingGraph(mapped);
-    applicationEventPublisher.publishEvent(new ResourceCreatedEvent(persisted.getId()));
+    addExistingEdges(persisted);
+    applicationEventPublisher.publishEvent(new ResourceCreatedEvent(persisted));
     return resourceDtoMapper.toDto(persisted);
   }
 
   @Override
   public Long createResource(org.folio.ld.dictionary.model.Resource modelResource) {
     var mapped = resourceModelMapper.toEntity(modelResource);
     var persisted = saveMergingGraph(mapped);
+    addExistingEdges(persisted);
```

Comment:
And here it's incoming modelResource, containing some piece of Work provided by marc4ld library, and this is a problem. Incoming work contains new instance while existed instance is not here. 
Yes, we could fix it adding incoming edge with existed Instance explicitly, like it's done below in addExistingEdges method.
But there are questions:
1) Do we lose only incoming pre-existed edges here? What about pre-existed outgoing edges?
2) What about other resource types? Let's say incoming ProviderEvent without a Place edge, while pre-existed ProviderEvent with a Place edge?
3) May be we should make it like general approach: refresh each incoming through marc4ld resource before saving it?